keps/sig-scheduling/20200114-multi-scheduling-profiles.md

```diff
@@ -244,19 +246,33 @@ the scheduler documentation.
 
 ### Test Plan
 
-TODO
+The following tests need to be in place:
 
-### Graduation Criteria
+- **Unit Tests**: For Component Config API conversion, validation and defaults, core scheduler and
+scheduler instantiation. Current tests that use a default scheduler (or default framework) should
+continue passing with no configuration changes.
+- **Integration tests**: Current tests with a default scheduler should continue passing with no
+configuration changes. We need new tests exercising more than one profile, in which:
+    - Each profile would favor specific nodes, so that we can verify assignment.
+    - Pods get binding events for the selected scheduler name.
+    - Pods that don't specify a scheduler name continue to be scheduled by the default profile.
+- **E2E tests**: A representative case from Integration tests will be replicated as E2E.
 
-##### Alpha -> Beta Graduation
+### Graduation Criteria
 
-TODO
+#### Alpha (v1.18):
 
-##### Beta -> GA Graduation
+These are the required changes:
 
-TODO
+- [ ] New `v1alpha2` ComponentConfig API, with conversion from `v1alpha1` and defaults.
+- [ ] New feature gate.
```

Comment:
@liggitt do we really need a feature gate? it looks to me like this is mostly a refactoring, and I am not sure a feature gate helps too much in gating a refactoring.

Reply:
I removed the feature gate and added a note in the following paragraph.